Makefile

```diff
@@ -20,6 +20,16 @@ else
 GOBIN=$(shell go env GOBIN)
 endif
 
+## Location to install dependencies to
+LOCALBIN ?= $(shell pwd)/bin
+$(LOCALBIN):
+	mkdir -p $(LOCALBIN)
+
+CONTROLLER_GEN ?= $(LOCALBIN)/controller-gen
+
+## Tool Versions
+CONTROLLER_TOOLS_VERSION ?= v0.9.2
```

Comment:
We have been using version v0.10.0 in the main branch (and using it to generate the manifests) since a few weeks.

Reply:
Fixed.